geometry_msgs/Point pt1
geometry_msgs/Point pt2
